spec/factories.rb

```diff
@@ -22,10 +22,17 @@
     todo_list
   end
 
-  factory :tag do |t|
+  factory :tag, class: ActsAsTaggableOn::Tag do |t|
```

Comment:
Prawdopodobnie również factories dla tagów nie będą potrzebne - łatwiej to zrobić przez `todo_item.tags.add("name")` jesli faktycznie będziesz potrzebowała. Staraj się nie zagłębiać w to jak wewnętrznie działa gem w trakcie testowania - to się czasem lubi zmienić i wtedy z niewiadomego powodu się testy posypią. Aczkolwiek wiedzieć dla wiedzenia warto.